### PR TITLE
add tests for strings

### PR DESCRIPTION
**Pull request by venky333**
* Added tests for strings

### CODE REVIEW

File: strings_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+type Company struct {
+	Name string
+	Address Address
+	Country string
+}
+
+var (
+	response       = "gitlab.Person{Name:\"name\", Age:16, NickNames:[\"n\" \"a\" \"m\" \"e\"], Address:gitlab.Address{Street:\"\", City:\"\", Province:\"\", Country:\"\"}}"
+	emptySliceResp = "gitlab.Person{Name:\"name\", Age:16, Address:gitlab.Address{Street:\"\", City:\"\", Province:\"\", Country:\"\"}}"
+)
```
Comment (svanharmelen) [inline strings_test.go:32]: Can these be formatted for readability and maybe use backtics instead of escaping the quotes? Or does that break the tests?

Comment (svanharmelen) [reply strings_test.go:32]: And since all of these are used only once, place declare them inside the test instead of making these globals. Thanks!

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: strings_test.go (hunk 1 above)
Comment (venky333) [reply strings_test.go:32]: * backticks will break the tests. So not changing anything there.
* declared them inside test.

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

**Pull request merged**